app/src/DTO/LdapEntryDTO.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\DTO;
+
+use RuntimeException;
+use Symfony\Component\Ldap\Entry;
+
+class LdapEntryDTO
+{
+    //TODO Create attrribute associate with LdapEntry
+    //TODO Create getter and setter
+
+    /**
+     * @var string
+     */
+    public $distinguishedNames;
+
+    /**
+     * @var array
+     */
+    public $attributes;
+
+    public function setdn($distinguishedNames): void
+    {
+        $this->dn = $distinguishedNames;
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
```

Comment:
What's this ??! There is no name attribute in `LdapEntryDTO` 😕 
I'm even surprised quality tools didn't pick this up.